### DISCUSSION TITLE
Is it OK to reuse a translation tiddler?

### DISCUSSION THREAD

**Discussion by tw-FRed** [TiddlyWiki/TiddlyWiki5#7632] [Q&A] [answered]
The `$:/language/SideBar/Contents/Caption` tiddler is used in TW language plugins to translate the **Contents** sidebar tab label.
Would it be OK to reuse the same tiddler to translate another Contents related UI element? For instance the Contents menu of the MenuBar plugin?

Fred

**Comment by pmario** [comment, marked as answer]
Usually every core element has their own translation strings. So for the core I would say no. For your own plugins it would be important, that it would not be changed soon. 

So if you have a look at, when that string was [changed the last time](https://github.com/Jermolene/TiddlyWiki5/blame/1a92fd5dc0455659c0529f0169f429f00443223d/core/language/en-GB/SideBar.multids#L5C27-L5C27) it's 9 years ago. So IMO it would be safe to use it. 

2 years ago I did upload a [translateabl-plugins](https://github.com/pmario/TiddlyWiki5/tree/translateable-plugins) branch to my repo.  It's now a 1000 commits behind master ;) I would have to review it and create a draft-PR. So the vercel-bot would be able to create a test-version.

Edit: Here's the old draft PR, which has no preview https://github.com/Jermolene/TiddlyWiki5/pull/5418

**Reply by linonetwo** [reply]
Great! I can't wait to use the same translation mechanism as the core. It will save lots of `<$list>` widget.

**Comment by Jermolene** [comment]
Hi @tw-FRed while @pmario is correct in general, the specific example you gave of the "Contents" tab and the "Contents" menu bar item does seem to be a case where the meaning is exactly the same, and so the string could be reused.
